src/api/v1/user.go

```diff
@@ -45,11 +49,14 @@ func getUser(c echo.Context) error {
 		// Lookup minecraft and discord in parallel
 		minecraftCh := make(chan resultMC)
 		discordCh := make(chan resultDiscord)
+		editionCh := make(chan resultEdition)
 		go func() { minecraftCh <- lookupMinecraftInfo(user.MinecraftID) }()
 		go func() { discordCh <- lookupDiscordInfo(user.DiscordID) }()
+		go func() { editionCh <- resultEdition{Edition: user.Edition()} }()
```

Comment:
getting the Edition doesn't involve any IO, but it does have a handful of small loops. No harm in giving it its own goroutine here?